src/modules/sync/advancedDiff.ts

```diff
@@ -0,0 +1,543 @@
+import {
+  ContentTypeElements,
+  ContentTypeModels,
+  ContentTypeSnippetModels,
+  TaxonomyModels,
+} from "@kontent-ai/management-sdk";
+import chalk from "chalk";
+import { readFileSync, writeFileSync } from "fs";
+import open from "open";
+import { resolve } from "path";
+
+import { SyncParams } from "../../commands/diff/diff.js";
+import { logError, logInfo } from "../../log.js";
+import {
+  isCountLimitation,
+  isDefaultValue,
+  isDependsOn,
+  isExternalIdReference,
+  isExternalIdReferenceArray,
+  isMaximumTextLength,
+  isObjectReference,
+  isObjectReferenceArray,
+  isValidationRegex,
+} from "../../utils/typeguards.js";
+import { RequiredCodename } from "../../utils/types.js";
+import { DiffModel, DiffObject } from "./types/diffModel.js";
+import {
+  AddIntoPatchOperation,
+  MovePatchOperation,
+  PatchOperation,
+  RemovePatchOperation,
+  ReplacePatchOperation,
+  ReplacePatchOperationValue,
+} from "./types/patchOperation.js";
+
+type DiffData = DiffModel & SyncParams;
```

Comment:
SyncParams? I don't think that's right

Reply:
me neither, guess I didn't think twice about it and copied what was in diff.ts the first time. fixed in both places.